cdap-ui/app/features/hydratorplusplus/services/create/stores/left-panel-store.js

```diff
@@ -146,7 +146,7 @@ class HydratorPlusPlusLeftPanelStore {
   }
 
   getPlugins(extension) {
```

Comment:
nit - there is a usage of this in another file that redundantly does `|| []` now. That can be removed.